### PR TITLE
build/configs/imxrt1050-evk, imxrt1020-evk: Fix APP_BINARY_SEPARATION issue

### PR DESCRIPTION
**Pull request by sangam-swami**
Booting fail with APP_BINARY_SEPARATION enable in few configs.
Add sdram settings and MM_REGION count for loadable elf apps,
SAM, wifi loadable configs to fix the issue.

Signed-off-by: Sangamanatha <sangam.swami@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [9b6049f28a47e6184c115310f902a51dbc717226] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [9b6049f28a47e6184c115310f902a51dbc717226] - Code Rule Check (C++) OK.

File: build/configs/imxrt1050-evk/loadable_elf_apps/defconfig [hunk 1]
```diff
@@ -281,8 +281,8 @@ CONFIG_BOOT_RUNFROMFLASH=y
 #
 # Boot Memory Configuration
 #
-CONFIG_RAM_REGIONx_START="0x20200000"
-CONFIG_RAM_REGIONx_SIZE="524288"
+CONFIG_RAM_REGIONx_START="0x20200000,0x80200000"
+CONFIG_RAM_REGIONx_SIZE="524288,31457280"
```
Comment (thapav) [inline build/configs/imxrt1050-evk/loadable_elf_apps/defconfig]: KIndly remove CONFIG_MM_PARTITION_HEAP from defconfig as this feature was added due to removal of partition manager module

**Comment by seinfra** [comment]
Target : [51ae7299d51c35206c325d5d4a892fc10cd12e07] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [51ae7299d51c35206c325d5d4a892fc10cd12e07] - Code Rule Check (C++) OK.

File: build/configs/imxrt1050-evk/loadable_elf_apps/defconfig (hunk 1 above)
Comment (sangam-swami) [reply build/configs/imxrt1050-evk/loadable_elf_apps/defconfig]: It was disabled by default, have removed now from defconfig

**Comment by sunghan-chang** [comment]
@sangam-swami Could you fix the wifi_loadable as well?

**Comment by sunghan-chang** [comment]
sam as well
```
Defconfig (build\configs\imxrt1020-evk\loadable_elf_apps):CONFIG_APP_BINARY_SEPARATION=y
Defconfig (build\configs\imxrt1020-evk\sam):CONFIG_APP_BINARY_SEPARATION=y
Defconfig (build\configs\imxrt1050-evk\loadable_elf_apps):CONFIG_APP_BINARY_SEPARATION=y
Defconfig (build\configs\imxrt1050-evk\wifi_loadable):CONFIG_APP_BINARY_SEPARATION=y
```

**Comment by sangam-swami** [comment]
> @sangam-swami Could you fix the wifi_loadable as well?

**Comment by sangam-swami** [comment]
> @sangam-swami Could you fix the wifi_loadable as well?

done

**Comment by sangam-swami** [comment]
> sam as well
> 
> ```
> Defconfig (build\configs\imxrt1020-evk\loadable_elf_apps):CONFIG_APP_BINARY_SEPARATION=y
> Defconfig (build\configs\imxrt1020-evk\sam):CONFIG_APP_BINARY_SEPARATION=y
> Defconfig (build\configs\imxrt1050-evk\loadable_elf_apps):CONFIG_APP_BINARY_SEPARATION=y
> Defconfig (build\configs\imxrt1050-evk\wifi_loadable):CONFIG_APP_BINARY_SEPARATION=y
> ```

done

**Comment by seinfra** [comment]
Target : [4829a43eda18ac5abcbffbaf60d9df4f231a6f86] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [4829a43eda18ac5abcbffbaf60d9df4f231a6f86] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [cb1fffeefbfdb43b01eeb5c590d83dcccd4d7c39] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [cb1fffeefbfdb43b01eeb5c590d83dcccd4d7c39] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [64764a4ad0bdf36ef2643046ef8fdf8486a76ed2] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [64764a4ad0bdf36ef2643046ef8fdf8486a76ed2] - Code Rule Check (C++) OK.

**Pull request merged**